pint/models/phase_jump.py

```diff
@@ -9,11 +9,13 @@
 
 
 class PhaseJump(TimingModel):
-    """This is a class to implement phase jumps 
+    """This is a class to implement phase jumps
     """
     def __init__(self):
         super(PhaseJump, self).__init__()
-        self.add_param(p.maskParameter(name = 'JUMP'))
+        # NOTE: it is dangours to put jump unit as second
```

Comment:
In a comment like this, please put some more explanation about why it is dangerous.  Someone looking at it should be able to understand what the issue is.  Link to an issue on github, if needed.